.cargo-husky/hooks/pre-commit

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh -x
```

Comment:
Upon checkout, is this guaranteed to have executable permission?

> Give an executable permission to the files (on *nix OS).

https://stackoverflow.com/questions/40978921/how-to-add-chmod-permissions-to-file-in-git

Reply:
My understanding is that cargo-husky should handle that. This file isn't used directly. It's "installed" to `.git/hooks` by cargo-husky upon `cargo test` in the `rethnet` crate.

Reply:
Oh I see now in the docs that you're expected to do that. I didn't do that here. I'll do that.

Reply:
Looks like it was already done, presumably because the `+x` was already set locally when i `git add`ed it:

```$ git ls-files -s .cargo-husky/hooks/pre-commit 
100755 8bbfeb0d4328f955308e1597697a3c3d45b7d270 0	.cargo-husky/hooks/pre-commit
```

"100755, which means it’s an executable file" ([source](https://git-scm.com/book/sv/v2/Git-Internals-Git-Objects))